manifests/params.pp

```diff
@@ -54,7 +54,10 @@
       # This option is not relevant for RedHat
       $sysconfig_resolvconf_integration = undef
 
-      $dnssec_enable = 'yes'
+      $dnssec_enable = $facts['os']['Family'] ? {
+        'RedHat' => if versioncmp($facts['os']['release']['major'], '8') >= 0 { undef } else { 'yes' },
+        default  => undef,
+      }
```

Comment:
You're already in the `RedHat` os family branch.
```suggestion
      $dnssec_enable = if versioncmp($facts['os']['release']['major'], '8') >= 0 { undef } else { 'yes' }
```

However, I do think that on EL8 it's still relevant. That has 9.11 and only 9.15 made it obsolete. So should it be this?
```suggestion
      $dnssec_enable = if versioncmp($facts['os']['release']['major'], '9') >= 0 { undef } else { 'yes' }
```

Reply:
I'd not fully realised that higher up I'd joined the RedHat branch, this was a great spot and helpful, thank you